bindings/python/tests/test_parse.py

```diff
@@ -32,3 +32,75 @@ def get_parsed_info(tmp_path, file_contents):
     )
 
     return parsed_info
+
+
+def test_return_type_composition(tmp_path):
```

Comment:
I'm sorry but I didn't understand what composition is being tested here

Reply:
Followed a suggestion: https://github.com/divmadan/pcl/pull/6#discussion_r457238713

Reply:
@aponza can you help me understand the `composition` under test here

Reply:
This test is for how the return type is composed, i.e. the structure of the data structure. A dictionary with lists, basically. Better names are welcome.

Reply:
I understand now. `test_parsed_info_{dict,struct}` or something similar?

Reply:
`test_parsed_info_class_structure`?

Reply:
Just `test_parsed_info_structure`?